.github/actions/upload-artifact/action.yml

```diff
@@ -17,3 +17,63 @@ runs:
         name: ${{ inputs.name }}
         path: ${{ inputs.path }}
         retention-days: 14
+
+    - id: get_commit_hash
+      run: echo "::set-output name=hash::$(git rev-parse HEAD)"
+      shell: bash
+
+    - name: Strip Binary
+      run: |
+        for file in ${{ inputs.path }}
+        do
+          strip "$file"
+        done
+      shell: bash
+
+    - name: Store artifact size
+      shell: bash
+      run: |
+        git_hash=${{ steps.get_commit_hash.outputs.hash }}
+        actual_size=$(du -sb ${{ inputs.path }} | cut -f1)
```

Comment:
`b` is not a valid flag for `du`, what was it intended to do?

Reply:
I was trying to look for a flag that allows getting byte size but this isn't a flag on the macos.

Reply:
There's `B` for block size